llama_stack/apis/inference/inference.py

```diff
@@ -246,4 +255,7 @@ async def embeddings(
         self,
         model: str,
         contents: List[InterleavedTextMedia],
+        sampling_params: Optional[SamplingParams] = SamplingParams(),
```

Comment:
these parameters you added don't seem relevant for a model which produces an "embedding" of a content. Is there a reason you added these?

Reply:
https://github.com/ollama/ollama/blob/main/docs/api.md#generate-embeddings

ollama's API seems to take this. I could take it out if it makes the API simpler

Reply:
I think only the `truncate` option could perhaps make sense. Let's not add any of these parameters for now -- typically one doesn't use them for embeddings at all.